src/base/memory_monitor_cl.hpp

```diff
@@ -26,6 +26,13 @@
 
 #include "memory_monitor_common.h"
 
+typedef enum mmon_module_option
+{
+  MMON_MODULE_DETAIL,
+  MMON_MODULE_BRIEF,
+  MMON_MODULE_DEFAULT
```

Comment:
For other developer who will develop this module, add description(comment) for each case.